src/modules/API.js

```diff
@@ -0,0 +1,29 @@
+const createGame = async (gameName) => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ name: gameName }),
+  });
+
+  const data = await response.json();
+  const key = data.result.split('ID: ')[1]; // Extract the game ID (key) from the response
+  return key;
+};
+
+const getAPI = async () => {
+  const data = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/m2U5S10HVbe4Z3PQkXWr/scores/');
+  const result = await data.json();
+  return result;
+};
+
+const setAPI = async (data) => {
+  const fetchData = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/m2U5S10HVbe4Z3PQkXWr/scores/', {
+    method: 'POST',
```

Comment:
- [ ] It would be great if you save your API URL into a variable in a file and then you can import it wherever you wanna use it to follow the DRY principal and this you won't make mistakes writing API's at different places because sometimes it happens you write the same API different places and makes some typos or something messy into the API URLs which breaks that specific feature or the whole app 😨 So it would be nice if you store this URL at some place in variable and use it wherever you want by importing it to follow the JavaScript best practices DRY 😊 👍